controllers/games.controllers.js

```diff
@@ -0,0 +1,10 @@
+const {selectCategories} = require('../models/games.models')
+
+exports.getCategories = (req, res) => {
+  selectCategories().then((category) => {
+    res.status(200).send({ category: category });
+  });
```

Comment:
Catch block needed for all controllers please. 500 errors will need to be handled, so an error handler to go with it also.